sgkit/tests/test_popgen.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import msprime
+import numpy as np
+
+from sgkit import ts_to_dataset, diversity, divergence
+
```

Comment:
Let's bring the definition of ``ts_to_dataset`` in here for now - we can move it into a better location later once we get an idea of where and how its used.